internal/gazelle.bash.in

```diff
@@ -29,10 +30,15 @@ GOTOOL=@@GOTOOL@@
 function find_runfile {
   local runfile=$1
   if [ -f "$runfile" ]; then
-    readlink "$runfile"
+    readlink "$runfile" || echo "$runfile"
```

Comment:
The addition of `echo "$runfile"` in reality is not necessary for the end-user running this code since bazel will always symlink this file if the -f test succeeds. 

However, the addition of echo is not incorrect in my opinion, and facilitates testing this script when symlinks are disabled. i.e. this allows someone to boot into windows and get all the tests pass despite not having symlinks on their machine.